diagnostic.md

```diff
@@ -76,7 +76,7 @@ var groceryList = [
 
 What is the value of `groceryList[2]`?
 
-_Replace this text with your answer._
+The value of `groceryList[2]` is `a container of milk`
```

Comment:
Remember that the index of an array starts at 0. So since you're looking for the 3rd value in the array, the answer is 'a stick of butter'